ezyrb/cvt.py

```diff
@@ -90,10 +95,56 @@ def _compute_simplex_volume(simplex_vertices):
 
 		return volume
 
-	def compute_errors(self):
+	def set_error_estimator(self, func):
+		"""
+		Allows to user to specify a custom function to estimate error in
+		loo compute error procedure.
+
+		:param function func: the function to use as error estimator: it have
+			to take as input a 1D numpy array and return a float
+		"""
+		self.__error_estimator_func = func
```

Comment:
Yeah, Luca shares good answer; i tipically prefer the double underscore because members become really private and you cannot access them outside of class (as C++). But as you want, no problem for me :)